### DISCUSSION TITLE
LeftNavbar click override

### DISCUSSION THREAD

**Discussion by dirkieman** [ColorlibHQ/AdminLTE#4609] [Q&A] [answered]
Hi, Please help me:
Is it possible to fire a custom JS function/method, when an item in the left navbar is clicked?
I want to load the selected page self, as I need to do some checks etc

**Comment by dfsmania** [comment, marked as answer]
You may use `Javascript`. The next example assumes you have assigned an `id` to the mentioned item on the left navbar:

```js
document.getElementById("theItemId").addEventListener("click", function(event)
{
    // You can prevent the default click behaviour uncommenting next line.
    // event.preventDefault()

    // Then, implement your logic to handle the click event on the item.
    // ...
});
```

**Reply by dirkieman** [reply]
thx Diego..
